svgr-template.js

```diff
@@ -0,0 +1,34 @@
+const template = (variables, { tpl }) => {
+  return tpl`
+${variables.imports};
+import {useTheme} from '@emotion/react'
+
+${variables.interfaces};
+
+const ${variables.componentName} = (${variables.props}) => {
+  const theme = useTheme()
+  const {
+    color,
+    size,
+    sizeH,
+    sizeW,
+    height: heightProp,
+    width: widthProp,
+    fill: fillProp,
+    ...other
+  } = props
+
+  const height = heightProp ?? theme.iconSizes[sizeH ?? size ?? 'l']
+  const width = heightProp ?? theme.iconSizes[sizeW ?? size ?? 'l']
```

Comment:
also do we actually need separate height and width? is this intentional to keep it square? if so, a comment would be good

Reply:
yeah @DejayJD added it, i think for things like the horizontal kebab. but agreed in those cases are we not trying to keep in square?

Reply:
I added the `sizeH/W` specifically because of logos, which also use this component/template. The logo was using preset sizes but only vertically, so I added them